packages/aws-cdk-lib/aws-ecs/lib/fargate/fargate-task-definition.ts

```diff
@@ -77,6 +78,17 @@ export interface FargateTaskDefinitionProps extends CommonTaskDefinitionProps {
    * @default - Undefined.
    */
   readonly runtimePlatform?: RuntimePlatform;
+
+  /**
+   * The process namespace to use for the containers in the task.
+   *
+   * Only supported for tasks that are hosted on AWS Fargate if the tasks
+   * are using platform version 1.4.0 or later (Linux).
```

Comment:
Is there a way to add a check for the platform version?

Reply:
Yup 👍 Thanks for the feedback! I missed that